playwright/e2e/suites/inventory.disable.ts

```diff
@@ -0,0 +1,48 @@
+import { test, expect } from '@playwright/test';
```

Comment:
the filename shows `disable`, does that mean these tests are skipped?

Reply:
Yep